subgraphs/_reference_/src/sdk/protocols/bridge/pool_snapshot.ts

```diff
@@ -0,0 +1,321 @@
+import { ethereum, Bytes } from "@graphprotocol/graph-ts";
```

Comment:
again, I have been using camelCase for ts files

Reply:
But here it says underscores: https://google.github.io/styleguide/tsguide.html#identifiers-imports

🫤

Reply:
Haha damn. Which one do you prefer? Let’s settle on one and add it to the CI

Reply:
We should go with the google guide imo. So you are good, the ci will catch the file names as we edit those files